src/views/virtualmachines/details/VirtualMachineNavPage.tsx

```diff
@@ -25,7 +24,7 @@ const VirtualMachineNavPage: React.FC<VirtualMachineDetailsPageProps> = ({
   const pages = useVirtualMachineTabs();
   return (
     <>
-      <VirtualMachineNavPageTitle vm={vm} />
+      <VirtualMachineNavPageTitle vm={vm} namespace={namespace} />
```

Comment:
i dont think you need namespace, you can use VM?.metadata?.namespace in ...PageTitle